Code/src/main/java/nl/utwente/group10/ui/components/blocks/FunctionBlock.java

```diff
@@ -1,373 +0,0 @@
-package nl.utwente.group10.ui.components.blocks;
```

Comment:
Moving classes makes it a bit hard to see what's changed. Could you propose a PR with just the move and no other changes?